experimenter/experimenter/nimbus-ui/src/components/PageResults/index.test.tsx

```diff
@@ -213,6 +213,46 @@ describe("PageResults", () => {
     expect(within(analysisBasisSelectParent).getByText(EXPOSURES_BASIS));
   });
 
+  it("displays the reference branch options properly", async () => {
+    render(<Subject mockAnalysisData={mockAnalysisWithExposures} />);
+
+    expect(screen.getByText("Analysis Basis"));
+    const referenceBranchSelectParent = screen.getByTestId(
+      "reference-branch-results-selector",
+    );
+
+    // both exist
+    expect(within(referenceBranchSelectParent).getByText("control"));
```

Comment:
You don't need the `expect(...)` wrapping here. You can just call within(xyz).getByText(abc) as it will assert if it does not find the element.